tests/test_other.py

```diff
@@ -7810,6 +7810,9 @@ def test_ioctl_window_size(self):
   def test_fd_closed(self):
     self.do_other_test('test_fd_closed.cpp')
 
+  def test_fd_pipe(self):
+    self.do_other_test('test_fd_pipe.c')
```

Comment:
We already have several pipe tests.. could just just modify `tests/unistd/pipe.c` instead perhaps?